src/App.js

```diff
@@ -1,23 +1,33 @@
-import logo from './logo.svg';
+import React, {useLayoutEffect, useState} from 'react';
```

Comment:
how is this being used?

Reply:
I delete useLayoutEffect because I don't use this function or method anymore.
I am using useState method to change the margin-left between 0-250px when NavBar pops out.